reference/generators/markdown.rst

```diff
@@ -10,7 +10,7 @@ consume them using different build systems.
 
 .. code-block:: bash
 
-    $ conan install poco/1.11.1@ --generator markdown
+    $ conan install poco/1.10.1@ --generator markdown
```

Comment:
why the downgrade?

Reply:
Because the screenshot is for poco/1.10.1